harness/determined/cli/master.py

```diff
@@ -89,19 +97,60 @@ def logs(args: Namespace) -> None:
         print(format_log_entry(response.logEntry))
 
 
+def parseLogLevel(level: str) -> bindings.v1LogLevel:
```

Comment:
this shouldn't be needed. python provides automatic support for turning strings into enums. 
`bindings.v1LogLevel["INFO"]`

Reply:
also, for future reference, functions should be camel-cased, bindings is just weird :)

Reply:
done!